Core/HLE/sceMpeg.cpp

```diff
@@ -1694,7 +1695,11 @@ static int sceMpegGetAtracAu(u32 mpeg, u32 streamId, u32 auAddr, u32 attrAddr)
 		Memory::Write_U32(0, attrAddr);
 	}
 
-	DEBUG_LOG(ME, "%x=sceMpegGetAtracAu(%08x, %08x, %08x, %08x)", result, mpeg, streamId, auAddr, attrAddr);
+	if (result == 0) {
+		DEBUG_LOG(ME, "%x=sceMpegGetAtracAu(%08x, %08x, %08x, %08x)", result, mpeg, streamId, auAddr, attrAddr);
```

Comment:
Might be nice to change this to `hleLogSuccessI` / `hleLogWarning` while changing it...

-[Unknown]

Reply:
Since we're doing a hleDelayResult below, I'm gonna leave it as is for now. I'm switching back to DEBUG.